src/components/SearchResults.js

```diff
@@ -11,14 +11,22 @@ function SearchResults({ results }) {
             <th scope="col">FirstName</th>
             <th scope="col">Surname</th>
             <th scope="col">Room Id</th>
-            <th scope="col">`check in date</th>
-            <th scope="col">check out date</th>
+            <th scope="col">Check In Date</th>
+            <th scope="col">Check Out Date</th>
           </tr>
         </thead>
         <tbody>
           {results.map((person, index) => {
+            const [select, setSelect] = useState(true);
+            function handleClick() {
+              setSelect(select => !select);
```

Comment:
Using `!select` rather than `if(select) {return false;} else {return true;}`, for example, is good! Terse, but also readable.